docs/pages/access-controls/sso/oidc.mdx

```diff
@@ -193,6 +193,28 @@ spec:
        roles: [ "access" ]
 ```
 
+### Optional: Max Age
+
+Teleport supports setting the `max_age` field since version 13.3.7 to control the
```

Comment:
```suggestion
Teleport has supported setting the `max_age` field since version 13.3.7 to control the
```
To agree grammatically with "since version...".